### PR TITLE
chore:Added changes to use scripts instead of community Github actions

### PR DESCRIPTION
**Pull request by ravali-rimmalapudi**
# Fixes #

Conversion of community Github actions into scripts.

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli-core/blob/main/CONTRIBUTING.md) and my PR follows them
- [ ] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -72,31 +72,27 @@ jobs:
...
-          token: ${{ secrets.REPO_ACCESS_TOKEN }}
-          repo: twilio/twilio-cli
-          ref: ${{ github.event.inputs.cli-branch }}
-          inputs: '{ "change-log": ${{ toJSON(needs.update-api-specs.outputs.change-log) }}, "version-type": "${{needs.update-api-specs.outputs.version-type}}", "homebrew-branch": "${{github.event.inputs.homebrew-branch}}", "homebrew-prerelease": "${{github.event.inputs.homebrew-prerelease}}" }'
+        run: node .github/scripts/trigger-workflow.js
+        env:
+          REPO_ACCESS_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
+          WORKFLOW_NAME: 'release.yml'
+          BRANCH_NAME: ${{github.event.inputs.cli-branch}}
+          REPO_NAME: LakshmiRavali/twilio-cli
```
Comment (shrutiburman) [inline .github/workflows/release.yml:97]: To be updated later?

**Review by shrutiburman — APPROVED** [review]
Please address the `REPO_NAME` comment, rest looks good to go. :)

File: .github/workflows/release.yml (hunk 1 above)
Comment (ravali-rimmalapudi) [reply .github/workflows/release.yml]: Thanks for pointing. Updated.

**Review by shrutiburman — APPROVED** [review]
🚀 ✨

**Pull request merged**